plugins/inputs/riemann_listener/riemann_listener.go

```diff
@@ -181,7 +181,7 @@ func (rsl *riemannListener) read(conn net.Conn) {
 			}
 		}
 
-		messagePb := &riemangoProto.Msg{}
+		messagePb := &riemanngoproto.Msg{}
```

Comment:
Is there a reason you change the spelling? The diff would be much smaller if you change the import.